review-tools/ghmerge

```diff
@@ -29,8 +29,19 @@ Examples:
     exit 9
 }
 
+function check_tools {
+    which jq >/dev/null 2>&1
```

Comment:
I've recently learned that if you want to be more posixly, you should use [`command -v`](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html) rather than `which`.

(as a matter of fact, using bash's `type` command has issued a warning on the matter lately...  that's how I learned)